### PR TITLE
Bug 1514006 - Add impression wrapper for Pocket sections

### PR DESCRIPTION
**Pull request by ncloudioj**

### CODE REVIEW

**Comment by ncloudioj** [comment]
Note: for MVP, this implementation is almost the same as the current Pocket impression stats. We might want to add more sophisticated visibility checks (i.e. page scrolling) in the followup iterations.

**Comment by ncloudioj** [comment]
Hey @piatra, could you take a look at this?

**Comment by piatra** [comment]
All good I'll have a look, thanks :)

File: content-src/components/DiscoveryStreamBase/DiscoveryStreamBase.jsx [hunk 1]
```diff
@@ -17,11 +18,18 @@ export class _DiscoveryStreamBase extends React.PureComponent {
       case "CardGrid":
         return (<CardGrid feed={component.feed} />);
       case "Hero":
-        return (<Hero
-          title={component.header.title}
-          feed={component.feed}
-          style={component.properties.style}
-          items={component.properties.items} />);
+        const feed = this.props.DiscoveryStream.feeds[component.feed.url];
+        const iterms = component.properties.items;
```
[the remark this answers is not in the document]
Comment (ncloudioj) [reply content-src/components/DiscoveryStreamBase/DiscoveryStreamBase.jsx]: Oops, nice catch.

**Comment by ncloudioj** [comment]
Thanks! @piatra 

The reason of getting a 20-item impression ping is because that the `component.properties.items` is currently unset in the test endpoint, already confirmed with Mathijs and Gavin about this field, they will provide and enforce it for Hero, CardGrid, and List in the next endpoint release.

**Comment by k88hudson** [comment]
@ncloudioj can you add steps for QA to verify this to the bug?

**Comment by ncloudioj** [comment]
@k88hudson Sure, will do.

**Pull request merged**